es6.md

```diff
@@ -145,3 +155,37 @@ interface ClassExpression <: Class, Expression {
     type: "ClassExpression";
 }
 ```
+
+# Modules
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Declaration {
```

Comment:
If we avoid making this and import extend from declaration, then not only are we following the specs definition of declaration, but the types accepted for the declaration field just says `declaration`